src/options.cpp

```diff
@@ -52,6 +53,8 @@ std::vector<OptionParser::option_t> yarpgen::OptionParser::opt_codes{
      OptionParser::printVersion},
     {"-s", "--seed", true, "Pass a predefined seed", "Err",
      OptionParser::parseSeed},
+    {"", "--std", true, "Language standard of the test",
```

Comment:
Would be good to mention here what are the options and what is the default.

Reply:
I added comments and a better mechanism for heuristics like this.